src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1489,11 +1489,33 @@ details.rustdoc-toggle > summary.hideme > span {
 }
 
 details.rustdoc-toggle > summary::before {
-	content: "[+]";
+	content: "Expand";
 	font-weight: 300;
 	font-size: 0.8em;
 	letter-spacing: 1px;
 	cursor: pointer;
+	width: 17px;
+	height: max(17px, 1.1em);
+	background: data-url(plus-17x17.png) no-repeat top left;
```

Comment:
Why not simply using the image directly instead of going through a `data-url`?

Reply:
Right now, the CSS file never links to images. This is probably because the images have version numbers (plus.png becomes plus1.55.0.png), so it would require patching the style sheet either way.

Reply:
Ah right, my PR doing that hasn't been merged yet. Take a look at how I did: https://github.com/rust-lang/rust/pull/86892/files#diff-39d31d43319757bdd745b6b58f1d8593af4d28454f8ad0259b802bd776898e58R178-R203

That will allow you to simply add the background url from the rust side directly.